common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -526,24 +524,43 @@ def test_download_transcript_button_works_correctly(self):
         # check if we can download transcript in "srt" format that has text "00:00:00,260"
         self.assertTrue(self.video.downloaded_transcript_contains_text('srt', '00:00:00,260'))
 
-        # select the transcript format "txt"
-        self.assertTrue(self.video.select_transcript_format('txt'))
-
-        # check if we can download transcript in "txt" format that has text "Welcome to edX."
-        self.assertTrue(self.video.downloaded_transcript_contains_text('txt', 'Welcome to edX.'))
-
-        # open vertical containing video "B"
-        self.course_nav.go_to_vertical('Test Vertical-1')
-
-        # check if we can download transcript in "txt" format that has text "Equal transcripts"
-        self.assertTrue(self.video.downloaded_transcript_contains_text('txt', 'Equal transcripts'))
-
         # open vertical containing video "C"
         self.course_nav.go_to_vertical('Test Vertical-2')
 
         # menu "download_transcript" doesn't exist
         self.assertFalse(self.video.is_menu_present('download_transcript'))
 
+    def test_download_txt_transcript_link_works_correctly(self):
+        """
+        Scenario: Download 'txt' transcript link works correctly.
+        Given the course has Video components A and B in "Youtube" mode
+        And I have defined downloadable transcripts for the videos
+        Then I can download a transcript for Video A in "txt" format
+        """
+
+        data_a = {'sub': '3_yD_cEKoCk', 'download_track': True}
```

Comment:
Instead of duplicating this whole test point, use ddt to pass in the transcript type.